lib/presenter.rb

```diff
@@ -0,0 +1,45 @@
+class Presenter
+  def initialize(displayer:, clear:)
+    @displayer = displayer
+    @clear = clear
+  end
+
+  def present(board:, message:)
```

Comment:
To fully take advantage of abstracting out `Presenter`, it would be useful if `present()` just straight up took a `Board` object and a `Message` object rather than the flattened board and a message string.

Reply:
I agree! However, I am focused on completing a few new features this week and do not believe that the value added by passing a message object (or potentially the entire game) will outweigh the need to finish these features. This would require another refactor. I have refactored presenter and game to pass a Board object and that has improved the intent of present and I believe leaves the codebase in a good spot to fully take advantage of presenter in my next iteration. Thank you for the suggestion!

Reply:
Sure, that sounds quite reasonable.